config/index.js

```diff
@@ -268,6 +272,31 @@ var conf = convict({
         env: 'BOUNCES_ENABLED'
       },
       complaint: {
+        doc: 'Tiers of max allowed complaints per amount of milliseconds',
+        default: {
+          0: ONE_YEAR
+        },
+        env: 'BOUNCES_COMPLAINT'
+      },
+      hard: {
+        doc: 'Tiers of max allowed hard bounces per amount of milliseconds',
+        default: {
+          // 0 are allowed in the past day.
+          // 1 is allowed in the past year.
+          0: ONE_DAY,
+          1: ONE_YEAR
+        },
+        env: 'BOUNCES_HARD'
+      },
+      soft: {
+        doc: 'Tiers of max allowed soft bounces per amount of milliseconds',
+        default: {
+          0: FIVE_MINUTES
+        },
+        env: 'BOUNCES_SOFT'
+      }
+      /*
+      complaint: {
```

Comment:
I read through the diff before submitting, but something always manages to sneak through.